src/math/LeastSquares.cc

```diff
@@ -137,15 +134,16 @@ class EigensystemSolver : public LeastSquares::Impl {
         _eig.compute(fisher);
         if (_eig.info() == Eigen::Success) {
             setRank(_eig.eigenvalues().reverse());
-            log.debug<5>("SelfAdjointEigenSolver succeeded: dimension=%d, rank=%d", dimension, rank);
+            LOGL_DEBUG("TRACE4.afw.math.LeastSquares",
+                "SelfAdjointEigenSolver succeeded: dimension=%d, rank=%d", dimension, rank);
```

Comment:
I don't there's a need for the `TRACE4` prefix here; as long as there was only one level used in this file with this particular logging name, I don't think it matters if there were other levels used in `afw.math`.